clouddriver-kubernetes-v2/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/handler/KubernetesDeploymentHandler.java

```diff
@@ -31,9 +32,7 @@
 import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest.KubernetesKind;
 import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest.KubernetesManifest;
 import com.netflix.spinnaker.clouddriver.model.Manifest.Status;
-import io.kubernetes.client.models.V1beta2Deployment;
-import io.kubernetes.client.models.V1beta2DeploymentCondition;
-import io.kubernetes.client.models.V1beta2DeploymentStatus;
+import io.kubernetes.client.models.*;
```

Comment:
Small nit, we try to be explicit about dependencies and avoid using `*` in imports.

Reply:
done ✅